src/Core/src/Platform/Android/PlatformVersion.cs

```diff
@@ -7,8 +8,10 @@ public static partial class PlatformVersion
 	{
 		public static bool IsAtLeast(BuildVersionCodes buildVersionCode) => OperatingSystem.IsAndroidVersionAtLeast((int)buildVersionCode);
 
+		[SupportedOSPlatformGuard("android28.0")]
 		public static bool IsAtLeast(int apiLevel) => OperatingSystem.IsAndroidVersionAtLeast(apiLevel);
 
+		[SupportedOSPlatformGuard("android29.0")]
 		public static bool Supports(int platformApi) => OperatingSystem.IsAndroidVersionAtLeast(platformApi);
```

Comment:
I don't understand this and the one above, you can still pass in arbitrary `platformApi` values, why is it marked as `SupportedOSPlatformGuard("android29.0")]`?

Reply:
> I don't understand this and the one above, you can still pass in arbitrary platformApi values, why is it marked as SupportedOSPlatformGuard("android29.0")]?

I checked all usages and annotated with the highest OS version, though yes it is not optimal, maybe i should create static readonly fields for each OS version in this type and reuse it